### PR TITLE
DTGB-214: Fixed the letter spacing of buttons.

### PR DESCRIPTION
**Pull request by meirege**
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] I have read the **CONTRIBUTING** document.
- [x] I have updated the documentation accordingly.
- [x] I have updated the style guide CHANGELOG accordingly.
- [ ] I have updated the gent_base CHANGELOG accordingly.

### CODE REVIEW

**Review by Jeroen005 — APPROVED** [review]
hoewel ik de toegevoegde waarde in vraag stel :P

**Comment by meirege** [comment]
@Jeroen005 Jup, maar was op expliciete vraag van design ;)

File: styleguide/components/21-atoms/buttons/_buttons.scss [hunk 1]
```diff
@@ -19,6 +19,7 @@
   height: 1.9rem;
   padding: .6rem 1.3rem;
   font-size: .6rem;
+  letter-spacing: .4px;
```
Comment (delrueba) [inline styleguide/components/21-atoms/buttons/_buttons.scss:22]: use em

File: styleguide/components/21-atoms/buttons/_buttons.scss [hunk 2]
```diff
@@ -33,6 +34,7 @@
   height: 2.2rem;
   padding: .6rem 1.8rem;
   font-size: .7rem;
+  letter-spacing: .5px;
```
Comment (delrueba) [inline styleguide/components/21-atoms/buttons/_buttons.scss:37]: Use em.
There is no such thing as .5px.

File: styleguide/components/21-atoms/buttons/_buttons.scss [hunk 3]
```diff
@@ -47,6 +49,7 @@
   height: 2.5rem;
   padding: .7rem 2.2rem;
   font-size: .8rem;
+  letter-spacing: .6px;
```
Comment (delrueba) [inline styleguide/components/21-atoms/buttons/_buttons.scss:52]: Use em.
There is no such thing as .6px.

File: styleguide/components/21-atoms/buttons/_buttons.scss (hunk 1 above)
Comment (delrueba) [reply styleguide/components/21-atoms/buttons/_buttons.scss:22]: there is no such thing as .4px

**Pull request merged**